### PR TITLE
immudb 1.3.2

### PR DESCRIPTION
**Pull request by jeroiraz**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/immudb.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Immudb < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "64b809adca6510ce7879397985ca33f8e1de676bb5453f594b0b753314cb2483"
   end
 
-  depends_on "go" => :build
```
Comment (chenrui333) [inline Formula/immudb.rb:22]: Any upstream issue reference for the go1.19 support?

Comment (jeroiraz) [reply Formula/immudb.rb:22]: While we're using go1.18 to build official binaries, our CI pipeline builds and run tests using go1.19 as well.

Comment (iMichka) [reply Formula/immudb.rb:22]: Why go back to go 1.16 then?

Comment (jeroiraz) [reply Formula/immudb.rb]: thanks for the review. It was an unexpected change when bumping the formula, it seems the local repo had that change

**Review by chenrui333 — APPROVED** [review]

File: Formula/immudb.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/immudb.rb]: That makes total sense. Thanks!!

**Pull request closed without merging**